Benchmarks/BenchmarkData/AllFeatures.txt

```diff
@@ -1,3 +1,4 @@
+ 1
```

Comment:
I presume that this is some sort of check digit to distinguish between 2D and 3D for the checks. Is this documented somewhere?

Reply:
Yes, exactly. It is written/read in the writeTestValues and readTestValues methods of the TestDataClass and takes on the values  QUAD_MESH_TEST = 1 or HEX_MESH_TEST = 2. The reason is that there are a different number/names for 2D and 3D. Not sure it needs to be documented since it is used only within that class, and those files are not ones that one would edit by hand. But I am not opposed to it.

Reply:
It is quite "under the hood" so it is probably okay to not add it to the documents. It could be enough to add a comment about the different number of quality at line 222 of `Source/Mesh/MeshQualityAnalysis.f90`